terraform/gcp/k8s/demo-cert.yml

```diff
@@ -0,0 +1,9 @@
+apiVersion: networking.gke.io/v1
+kind: ManagedCertificate
+metadata:
+  name: demo-cert
+spec:
+  domains:
+    - sandbox.demo.juniper-cmi.dev
+    - irb.demo.juniper-cmi.dev
+    - demo.juniper-cmi.dev
```

Comment:
Moved these to their own cert object because I learned that managed certificates can only be _created_ and _deleted_. This means that, if we wanted to add a new domain, there would be about 40 minutes of downtime if we created all of the certificates for juniper-cmi in one file. Thus, I think it's best to create a separate certificate file for _each_ portal, that way that can be created/deleted and only effect that portal.